docs/axisman.rst

```diff
@@ -144,6 +144,26 @@ The output of the ``wrap`` cal should be::
 Note the boresight entry is marked with a ``*``, indicating that it's
 an AxisManager rather than a numpy array.
 
+Data access under an AxisManager is done based on field names. For example:
```

Comment:
In your example, you construct a new aman.  But in the flow of this documentation, an AxisManager already exists, which has been built up demonstrating the features and usage.  The existing one has enough stuff to demonstrate all the features you need to demonstrate (e.g. `dset.boresight.az` vs. `dset['boresight.az']`).  So please use that!  (And that should flow into the "To slice this object..." section, which again refers to the `dset` variable.)

Also this line for some reason renders in bold, in the docs... maybe add a blank line after?  Not sure how you got away without a double `::`...